src/content/docs/en/reference/image-service-reference.mdx

```diff
@@ -116,20 +125,20 @@ At build time for static sites and pre-rendered routes, both `<Image />` and `ge
 
 In dev mode and SSR mode, Astro doesn't know ahead of time which images need to be optimized. Astro uses a GET endpoint (by default, `/_image`) to process the images at runtime. `<Image />` and `getImage()` pass their options to `getURL()`, which will return the endpoint URL. Then, the endpoint calls `parseURL()` and passes the resulting properties to `transform()`.
 
-#### getConfiguredImageService
+#### getConfiguredImageService & imageServiceConfig
```

Comment:
@Jutanium - How do you feel about these two being combined into one heading? Ideally, I think we'd normally split them up, and define/explain each one separately.

Since you went through these examples extensively the first time around, do you have a suggestion for a sensible way to devote space to each one separately that doesn't force us to duplicate examples and make this long and terrible?

Not blocking, because the example below is entirely reasonable. But I do think it would be preferable if each were treated separately, if we had a good way of doing it.